<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>模板初始化</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextBesideIcon</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>41</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>41</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>路径：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edtPath"/>
       </item>
       <item>
        <widget class="QToolButton" name="btnOpenParentFolder">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../app.qrc">
           <normaloff>:/res/folder.ico</normaloff>:/res/folder.ico</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>项目名称：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edtName"/>
       </item>
       <item>
        <widget class="QToolButton" name="btnRun">
         <property name="text">
          <string>生成项目</string>
         </property>
         <property name="icon">
          <iconset resource="../app.qrc">
           <normaloff>:/res/run.png</normaloff>:/res/run.png</iconset>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btnOpenFolder">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../app.qrc">
           <normaloff>:/res/folder.ico</normaloff>:/res/folder.ico</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_2" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QListWidget" name="lvProjects">
         <property name="minimumSize">
          <size>
           <width>240</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>240</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTreeView" name="tvFile">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../app.qrc"/>
 </resources>
 <connections/>
</ui>
